### DISCUSSION TITLE
https://github.com/tsightler/ring-mqtt.git is not a valid add-on repository

### DISCUSSION THREAD

**Discussion by najama2** [tsightler/ring-mqtt#515] [Support Q&A]
Any idea when this will be fixed? Think something broke in the latest HA update. Thanks.

**Comment by sktaylortrash** [comment]
This isn't the repo for the addon? It is at https://github.com/tsightler/ring-mqtt-ha-addon

**Reply by najama2** [reply]
That's actually what I'm trying to add and it does not work. The error message above is simply copy/pasted from Home Assistant.

Hopefully an update can be made soon and it fixed.
![Screenshot 2022-12-04 221151](https://user-images.githubusercontent.com/80186088/205541415-2daf8a77-89f9-4647-8a24-2c6d3fff25e5.png)

**Reply by tsightler** [reply]
> Any idea when this will be fixed? Think something broke in the latest HA update. Thanks.

I can't reproduce this, and you provided no logs or details, so probably never.

**Comment by sktaylortrash** [comment]
@najama2  As a sanity check, I just tested adding it to a completely vanilla instance of 2022.11.5 The repo was added without issue for me as well

**Reply by Grey-Lancaster** [reply]
What version of HA are you running?  I tried with the wrong repo and got a very different error?

![image](https://user-images.githubusercontent.com/20055899/208955134-2ee22d2d-f319-49b4-b82f-c8cf23f34c0c.png)

**Comment by exr4361** [comment]
> Any idea when this will be fixed? Think something broke in the latest HA update. Thanks.

I was having this issue, restart Home Assistant and try again. I used the following repository link and it worked after: https://github.com/tsightler/ring-mqtt-ha-addon/
